_commands/editors/vi.md

```diff
@@ -0,0 +1,46 @@
+vi
+
+
+
+'vi' is a text editor for programmers, designed to facilitate optimal efficiency when writing code. 
+
```

Comment:
Your `.md` file isn't being listed in the reference section because you need some 'boilerplate' code from the `TEMPLATE.md` file located under the `_commands` folder.


All you have to do is change the top few lines to ...

~~~markdown
---
---

vi
----


'vi' is a text editor for programmers, designed to facilitate optimal efficiency when writing code. 
~~~